charts/vela-core/templates/addons/terraform-provider-alibaba.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: v1
+data:
+  initializer: |
+    apiVersion: core.oam.dev/v1beta1
+    kind: Initializer
+    metadata:
+      annotations:
+        addons.oam.dev/description: terraform provider (alibaba)
+        addons.oam.dev/name: terraform/provider-alibaba
+      name: terraform-provider-alibaba
+      namespace: vela-system
+    spec:
+      appTemplate:
+        spec:
+          components:
+          - name: alibaba-account-creds
+            properties:
+              apiVersion: v1
+              kind: Secret
+              metadata:
+                name: alibaba-provider-secret
+                namespace: vela-system
+              stringData:
+                credentials: |
+                  accessKeyID: [[ index .Args "ak" ]]
+                  accessKeySecret: [[ index .Args "sk" ]]
+                  securityToken: [[ index .Args "sts" ]]
```

```suggestion
                  accessKeyID: [[ index .Args "ALICLOUD_ACCESS_KEY" ]]
                  accessKeySecret: [[ index .Args "ALICLOUD_SECRET_KEY" ]]
                  securityToken: [[ index .Args "ALICLOUD_SECURITY_TOKEN" ]]
```

Terraform users is family with those changed variables.